app/frontend/wallet/shelley/shelley-transaction-planner.ts

```diff
@@ -286,57 +309,94 @@ export function computeTxPlan(
   // if we cannot create a change output with minimal ada we add it to the fee
   if (change.tokenBundle.length === 0 && change.coins < minimalChangeLovelace) {
```

Comment:
will this work for multiasset though? You can "burn" the ADA as fees, but that's not the case with tokens :thinking: